src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2927,6 +2933,120 @@ def _upgrade_single_nodepool_image_version(no_wait, client, resource_group_name,
     return sdk_no_wait(no_wait, client.upgrade_node_image_version, resource_group_name, cluster_name, nodepool_name)
 
 
+def aks_runcommand(cmd, client, resource_group_name, name, command_string="", command_files=None):
+    colorama.init()
+
+    mc = client.get(resource_group_name, name)
+
+    if not command_string:
+        raise CLIError('Command cannot be empty.')
```

Comment:
Please use [new error types](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md#error-type) in `azure-cli` code such as `ValidationError`.